etc/profiles.yml

```diff
@@ -0,0 +1,25 @@
+# The valid profile names are:
+# "microsoft-azure", "google-cloud-platform", "amazon-web-services", "default"
+# Default is "default" if no cloud environment detected
+# More details please see `man corosync.conf`
+
+default:
+  totem.crypto_hash: sha1
+  totem.crypto_cipher: aes256
+  totem.token: 5000
+  totem.consensus: 6000
```

Comment:
Let's default ` totem.consensus`, so that corosync automatically calculate a proper value at runtime.

Reply:
> Let's default ` totem.consensus`, so that corosync automatically calculates a proper value based on `totem.token` and so on at runtime.

Agree. Well, I do see the "pain" from the history for about , almost every

Reply:
Do you mean not write `consensus` in corosync.conf?
Agree